webpack/scenes/ContentViews/Table/tableDataGenerator.js

```diff
@@ -12,22 +12,28 @@ import LastSync from '../Details/Repositories/LastSync';
 
 export const buildColumns = () => [
   { title: __('Type'), cellFormatters: [expandable], transforms: [fitContent] },
-  __('Name'), __('Last published'), __('Last task'), __('Latest version'),
+  {
+    title: __('Name'),
+    transforms: [sortable],
+  },
+  __('Last published'),
+  __('Last task'),
+  __('Latest version'),
 ];
 
 const buildRow = (contentView) => {
   /* eslint-disable max-len */
   const {
     id, composite, name, last_published: lastPublished, latest_version: latestVersion, latest_version_id: latestVersionId,
-    latest_version_environments: latestVersionEnvironments, last_task: lastTask,
+    latest_version_environments: latestVersionEnvironments, last_task: lastTask, updated_at: updatedAt,
```

Comment:
The tooltip looks nice.

This updated_at doesn't seem like the correct value we want here..We need the ended_at of last_task..However the API doesn't return that information..We'll need to add it there..
For failed tasks, the ended_at might be null..So we can use start_date..Let me try adding both of those to the API for CV index..

We will also want to use this tooltip for other places LastSync component is used..

Reply:
We can add started_at here and use that instead of updated_at: https://github.com/Katello/katello/blob/901236e5fe931206600af10cd1212cfaf16bcda8/app/views/katello/api/v2/content_views/base.json.rabl#L20